NEWS.md

```diff
@@ -7,6 +7,8 @@ flux-accounting version 0.23.0 - 2023-04-04
 
 * plugin: clear queues on flux-accounting DB update (#334)
 
+* flux-accounting service: change Requires to BindTo (#338)
```

Comment:
Should this NEWS item go under a new 0.23.1 header?

Reply:
Oh, duh. 🤦‍♂️ I'm moving too fast. I'll fix this and add a new header